pkg/storage/jobevents.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package storage
+
+import (
+	"github.com/facebookincubator/contest/pkg/job"
+)
+
+// JobEventEmitterFetcher wraps all the job-related emitters and fetchers.
+type JobEventEmitterFetcher struct {
+	JobRequestEmitterFetcher
+	JobReportEmitterFetcher
+}
+
+// NewJobEventEmitterFetcher creates a new emitter/fetcher object for job events
+func NewJobEventEmitterFetcher() job.EventEmitterFetcher {
+	return JobEventEmitterFetcher{
+		JobRequestEmitterFetcher{},
```

Comment:
It's totally not important at all. Just I don't understand why it was required to explicitly write `JobRequestEmitterFetcher{},` and `JobReportEmitterFetcher{},`. Since it's not a pointer or/and interface it will be initialized with a zero value anyway.

Reply:
you are right, I was just reusing the existing style in the file, but it makes no sense and I'll change that. The whole function is not necessary, to be honest, but for this change I'm trying to touch as little as I can

Reply:
I'm removing it from other places